watertap/examples/flowsheets/RO_with_energy_recovery/RO_with_energy_recovery.py

```diff
@@ -212,11 +215,38 @@ def set_operating_conditions(
     # state variables
     m.fs.feed.properties[0].pressure.fix(101325)  # feed pressure [Pa]
     m.fs.feed.properties[0].temperature.fix(273.15 + 25)  # feed temperature [K]
+
+    # scaling
+    # set default property values
+
+    m.fs.properties.set_default_scaling(
+        "flow_mass_phase_comp", 1000 * flow_vol, index=("Liq", "H2O")
+    )
+    m.fs.properties.set_default_scaling(
+        "flow_mass_phase_comp", 1e-3 / flow_vol / salt_mass_conc, index=("Liq", "NaCl")
+    )
+
+    m.fs.feed.properties[0].flow_vol_phase["Liq"]
+    m.fs.feed.properties[0].mass_frac_phase_comp["Liq", "NaCl"]
+
+    # unused scaling factors needed by IDAES base costing module
+    # calculate and propagate scaling factors
+    iscale.calculate_scaling_factors(m)
+
+    iscale.set_scaling_factor(
+        m.fs.P1.control_volume.properties_out[0].flow_vol_phase["Liq"], 1
+    )
+    iscale.set_scaling_factor(m.fs.P1.work_fluid[0], 1)
+    iscale.set_scaling_factor(m.fs.RO.mass_transfer_phase_comp[0, "Liq", "NaCl"], 1e4)
+    iscale.set_scaling_factor(
+        m.fs.RO.feed_side.mass_transfer_term[0, "Liq", "NaCl"], 1e4
+    )
+
```

Comment:
any reason why this scale factors are being set **after** calling `calculate_scaling_factors` as opposed to **before** calling it?

Reply:
Nope... will update this